script/Arcueid/Day 4/04_00_ARC04_6_2.txt

```diff
@@ -1,7 +1,7 @@
 [sha:62ba0afe7179253e82b47c9989914ad42e56da54]{
 -- Page 10017, Offset 6725.
 -- 「……約束、したからな」
-"...I made a promise after all."
+"I made a promise, after all..."
```

Comment:
pause is really ugly in english. I say take it out.

moving ellipses to end is fine though